src/soter/soter_sign.c

```diff
@@ -14,7 +14,8 @@
 * limitations under the License.
 */
 
-#include <soter/soter.h>
+#include "soter_sign_ecdsa.h"
```

Comment:
why here we use relative to file's folder path instead `soter/soter_sign_ecdsa.h` ? it's third variant of imports in the code. do we have any reason for that?

Reply:
> do we have any reason for that?

No, that's just me not having enough attention. Thanks for noticing this, I'll review the includes once again. All paths should start from the 'project root'.